Task2/src/app.ts

```diff
@@ -0,0 +1,19 @@
+import './assets/styles/app-styles.css';
+
+import 'zone.js';
```

Comment:
Yep. You probably right , I just thought, that angular takes care about importing that.  Will check it tomorrow.